distribution/openrct2.d.ts

```diff
@@ -295,13 +295,13 @@ declare global {
     interface GameMap {
         readonly size: Coord2;
         readonly numRides: number;
-        readonly numThings: number;
+        readonly numEntities: number;
         readonly rides: Ride[];
 
         getRide(id: number): Ride;
         getTile(x: number, y: number): Tile;
-        getThing(id: number): Thing;
-        getAllThings(type: ThingType);
+        getEntity(id: number): Entity;
+        getAllEntities(type: EntityType);
```

Comment:
I thought our functions and methods used TitleCase?

Reply:
I am following JavaScript convention for this. Pretty much all JavaScript uses these rules. It is completely separate from our code base.